onnxscript/backend/onnx_export_test.py

```diff
@@ -70,8 +70,7 @@ def skip(pattern: str | Pattern, reason: str, *, condition: bool = True):
     ),
     skip(
         r"^test_optional_get_element_tensor",
-        "ORT Unable to create onnxruntime InferenceSession for executing .OptionalGetElement op with onnx model",
-        condition=ort.__version__[:4] == "1.14",
+        "ONNX backend test produces an invalid graph",
```

Comment:
What's this? Is this something needs to be fixed by ONNX?

Reply:
Yes. I filed an issue and linked above